### PR TITLE
Hookup couchdb and write seeding script

### PR DESCRIPTION
**Pull request by christulin**
## Trello Ticket
https://trello.com/c/6NhXN6pV/26-set-up-couchdb-and-write-seeding-script

## Purpose of Pull Request
This PR adds nano package to communicate with couchdb via node client, and creates a script to create db and seed with 10M unique records.

### CODE REVIEW

File: database/couchdb.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  try {
+    const response = await database.bulk({ docs });
+  } catch(e) {
+    console.error(e);
+  }
+
+};
+
+const createDB = async () => {
```
Comment (amosNash) [inline database/couchdb.js:31]: This looks good!

File: database/couchdb.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const faker = require('faker');
+const { config } = require('../config.js');
+const { listPhotos }  = require('./aws-s3.js');
+
+const seed = async (database) => {
+
+  const photos = await listPhotos();
+  const docs = [];
+
+  for (let i = 0; i < 998; i++) {
```
Comment (amosNash) [inline database/couchdb.js:13]: Why are you running this for loop until i is 998? Where did you get that number from?

File: database/couchdb.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
+      const name = photo.Key.split('.')[0];
+      const caption = faker.commerce.productName();
+      const is_primary = i % 5 === 0;
+      docs.push({ storage_url, name, caption, is_primary });
+    });
+  };
+
+  try {
+    const response = await database.bulk({ docs });
```
Comment (amosNash) [inline database/couchdb.js:24]: Way to use the bulk method to insert many records.

**Review by amosNash — COMMENTED** [review]
Looks good!

File: database/couchdb.js (hunk 2 above)
Comment (christulin) [reply database/couchdb.js:13]: I had just tested with 2, so filled in the rest with 998

**Pull request merged**